tiledb/sm/c_api/tiledb.h

```diff
@@ -6950,8 +6950,7 @@ TILEDB_EXPORT int32_t tiledb_fragment_info_get_non_empty_domain_var_from_index(
     void* end);
 
 /**
- * Retrieves the non-empty domain from a fragment for a given
- * dimension name. Applicable to var-sized dimensions.
+ * Retrieves the non-empty domain from a fragment for a given dimension index.
```

Comment:
Why is there a change here? This is indeed applicable only to var-sized dimensions, right?